### PR TITLE
Tooltip v2

### PR DESCRIPTION
**Pull request by mihai-macaneata**

### CODE REVIEW

**Comment by mihai-macaneata** [comment]
Don't merge it with master yet. Need to create the tooltip for at least one more component (maybe all of them)  and move replicated code in mixin

File: assets/js/components/mixins/TooltipMixin.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<style lang="less">
```
Comment (xlotlu) [inline assets/js/components/mixins/TooltipMixin.vue]: please rename to `WithTooltip.vue` and do
```js
import WithTooltipMixin from './mixins/WithTooltip';
```
instead

**Comment by mihai-macaneata** [comment]
I think this is ready for merge @xlotlu

**Comment by xlotlu** [comment]
@mihai-macaneata each filtering action adds 2 new "d3-tip" divs to the dom, one for sectors, one for beneficiaries.

**Pull request merged**